.eslintrc.js

```diff
@@ -24,7 +24,12 @@ module.exports = {
     'plugin:@typescript-eslint/stylistic-type-checked',
   ],
   parserOptions: {
-    sourceType: 'module',
+    allowAutomaticSingleRunInference: true,
+    cacheLifetime: {
+      // we pretty well never create/change tsconfig structure - so need to ever evict the cache
```

Comment:
*no need to ever

Reply:
https://github.com/typescript-eslint/typescript-eslint/pull/6367/files#diff-e2954b558f2aa82baff0e30964490d12942e0e251c1aa56c3294de6ec67b7cf5R44 😛